content/400-reference/200-api-reference/300-environment-variables-reference.mdx

```diff
@@ -113,6 +113,11 @@ PRISMA_CLI_QUERY_ENGINE_TYPE=binary
 ```env
 PRISMA_CLIENT_ENGINE_TYPE=binary
 ```
+If you are working in the [Prisma Data Platform](https://cloud.prisma.io) and want to use the [Prisma Data Proxy](/concepts/components/prisma-data-platform#prisma-data-proxy) to manage your database connections, you must edit the `.env` file to configure the `PRISMA_CLIENT_ENGINE_TYPE` environment variable. To do so, add the following line to your [`.env`file](/guides/development-environment/environment-variables#using-env-files):
```

Comment:
Re the `.env` file: https://github.com/prisma/docs/issues/2524#issuecomment-995247669

This applies to _all_ variables on this page. We should probably link to our "env variables" pages in the introduction instead of starting to explain for each one individually.